devops/providers/azure-devops/templates/infrastructure/azure-pipeline-build-stage.yml

```diff
@@ -61,8 +63,17 @@ jobs:
       inputs:
         version: '$(GO_VERSION)'
 
+    - task: Bash@3
+      name: SetupGitCredentialInjection
+      displayName: Configure Git to use PAT
+      condition: eq(${{ coalesce(config.overrideGitPAT, 'false') }}, 'true')
```

Comment:
I always get confused with boolean logic, but this may be simplified to something like this:
```bash
condition: coalesce(config.overrideGitPAT, 'false')
```

Reply:
This will look a bit different once the environment variable changes comes through, but yes -- it could definitely be simplified.